ledger/sandbox-classic/BUILD.bazel

```diff
@@ -444,15 +434,12 @@ server_conformance_test(
         "--concurrent-test-runs=1",  # sandbox classic doesn't scale well with concurrent tests (almost no effect on overall run time)
         "--timeout-scale-factor=2",  # sandbox classic is slow in general
         "--open-world",
+        "--static-time",
```

Comment:
How come it did work without this?

Reply:
Because sandbox-classic uses only participant deduplication and that always uses wall clock

Reply:
Then why is it needed now?

Reply:
The `DeduplicatingMixedClients` test case no longer runs in static time so we would either exclude it or actually be consistent and set the static time flag. 

This will be removed shortly as we are adding the static time as a feature descriptor.